### PR TITLE
Add merge conflict note to README file

### PR DESCRIPTION
**Pull request by KylerHansen**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,7 +19,8 @@ Use this instead of `git push` when you have a chain of several PRs that are tar
 1. Start with the base-most branch (the branch closest to your main dev branch) that needs to be updated.
 2. Make the necessary changes to that branch (rebase, commit amend, etc.).
 3. Run `git-vir push`.
-    - if you use a remote name besides `origin`, you can provide that as well with `git-vir <remote-name-here> push`
+    - If you use a remote name besides `origin`, you can provide that as well with `git-vir <remote-name-here> push`
```
Comment (electrovir) [inline README.md]:
```suggestion
    - If you use a remote name besides `origin`, you can provide that as well with `git-vir <remote-name-here> push`.
```

File: README.md [hunk 2]
```diff
@@ -19,7 +19,8 @@ Use this instead of `git push` when you have a chain of several PRs that are tar
...
+    - If you encounter merge conflicts, resolve the conflicts, run `git-rebase --continue` until rebase has finished. Then, run `git-vir push` to resume updating the chain.
```
Comment (electrovir) [inline README.md]:
```suggestion
    - If you encounter merge conflicts, resolve them as you would for a normal `git rebase` command. Then run `git-vir push` to resume updating the chain.
```

**Review by electrovir — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (KylerHansen) [reply README.md]: added

File: README.md (hunk 2 above)
Comment (KylerHansen) [reply README.md]: updated

File: README.md [hunk 3]
```diff
@@ -19,7 +19,8 @@ Use this instead of `git push` when you have a chain of several PRs that are tar
 1. Start with the base-most branch (the branch closest to your main dev branch) that needs to be updated.
 2. Make the necessary changes to that branch (rebase, commit amend, etc.).
 3. Run `git-vir push`.
-    - if you use a remote name besides `origin`, you can provide that as well with `git-vir <remote-name-here> push`
+    - If you use a remote name besides `origin`, you can provide that as well with `git-vir <remote-name-here> push`.
+    - If you encounter merge conflicts, resolve  them as you would for a normal `git rebase` command. Then, run `git-vir push` to resume updating the chain.
```
Comment (electrovir) [inline README.md:23]: Sorry, just one last tiny change:
```suggestion
    - If you encounter merge conflicts, resolve them as you would for a normal `git rebase` command. Then, run `git-vir push` to resume updating the chain.
```

**Review by electrovir — CHANGES_REQUESTED** [review]

**Comment by electrovir** [comment]
I'll fix this after merge

File: README.md (hunk 3 above)
Comment (electrovir) [reply README.md:23]: I'll fix this after merge

**Pull request merged**